samples/crypto/ecjpake/sample.yaml

```diff
@@ -0,0 +1,18 @@
+sample:
+  description: This app provides an example of EC J-PAKE
+  name: EC J-PAKE example
+tests:
+  sample.ecjpake:
+    tags: introduction psa
+    platform_allow: >
+      nrf5340dk_nrf5340_cpuapp_ns nrf5340dk_nrf5340_cpuapp
+      nrf9160dk_nrf9160_ns nrf9160dk_nrf9160 nrf52840dk_nrf52840
```

Comment:
- nrf52840dk_nrf52840

Reply:
it's there? (or am I missing something?)

Reply:
I'm just blind.